src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -234,6 +234,9 @@ private static unsafe bool InitializeConfigAndDetermineUsePortableThreadPool()
                 ? AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false)
                 : GetEnableWorkerTrackingNative();
 
+        internal static readonly bool EnableDispatchAutoreleasePool =
```

Comment:
Nit: Static fields should have `s_` prefix. It should be method or a property.

It would be nice if the naming is consistent with WorkerTracing switch right above this.

Reply:
The naming is consistent with the `EnableWorkerTracking` field naming on line 180 of this file.

Reply:
Ah ok, I have missed this. Ignore this comment.

Reply:
I got confused by the `GetEnableWorkerTracking` method that is not really necessary.